.github/assets/check_wasm.sh

```diff
@@ -74,6 +74,7 @@ exclude_crates=(
   reth-scroll-cli # tokio
   reth-scroll-node # tokio
   reth-scroll # tokio
+  reth-scroll-engine # CI fails on tokio but compiles locally
```

Comment:
this is very puzzling: keeps failing in CI with `compile_error!("Only features sync,macros,io-util,rt,time are supported on wasm.");`, while it compiles fine on a linux server with the wasm target, using the same toolchain.